Sources/Sentry/SentryFramesTracker.m

```diff
@@ -101,6 +122,17 @@ - (void)displayLinkCallback
             = 1 / (self.displayLinkWrapper.targetTimestamp - self.displayLinkWrapper.timestamp);
     }
 
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+    if (self.refreshRateTimestamps.count == 0
+        || self.refreshRateTimestamps.lastObject[@"refresh_rate"].doubleValue
+            != actualFramesPerSecond) {
+        [self.refreshRateTimestamps addObject:@{
+            @"timestamp" : @(self.displayLinkWrapper.timestamp),
+            @"refresh_rate" : @(actualFramesPerSecond),
```

Comment:
`l`: Why refresh rate? Shouldn't we call this `frame_rate` or `frames_per_second`?
```suggestion
            @"frame_rate" : @(actualFramesPerSecond),
```

Reply:
We want to know what the refresh rates are at all times. Most of the time it will be the same throughout the trace, but in the case of higher end devices with variable rates that can change like in low power scenarios, we want to know what the change was and when it changed.